AdSecCore/UnitHelpers.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+
+using OasysUnits;
+
+namespace AdSecCore {
+  public static class UnitHelpers {
+
+    public static T ParseToQuantity<T>(object goo, Enum units) {
+      if (goo is int intNumber) {
+        if (Quantity.TryFrom(intNumber, units, out var quantity)) {
+          return (T)quantity;
+        }
+      } else if (goo is double number) {
+        if (Quantity.TryFrom(number, units, out var quantity)) {
+          return (T)quantity;
+        }
+      } else if (goo is string txt) {
+        if (double.TryParse(goo.ToString(), out double result)) {
+          if (Quantity.TryFrom(result, units, out var quantity)) {
+            return (T)quantity;
+          }
+        } else if (Quantity.TryParse(typeof(T), txt, out var length)) {
```

Comment:
"out var lenght" pls change it to " out var result" or "out var quantity"

Reply:
yeah there is already a result there! I will think of something